.github/workflows-src/steps.lib.yml

```diff
@@ -1,20 +1,64 @@
-#@ def steps_genversion():
-name: "Generate Version"
+#@ def steps_build():
+name: Build CLI
 shell: bash
-run: state run generate-version
+run: state run build
 #@ end
 ---
-#@ def steps_build():
-name: Build
+#@ def steps_build_svc():
+name: Build Service
 shell: bash
-run: state run build
+run: state run build-svc
+#@ end
+---
+#@ def steps_build_tray_linux():
+name: Build Tray App (linux)
+shell: bash
+if: runner.os == 'Linux'
+run: |
+  sudo apt-get install libgtk-3-dev libappindicator3-dev -y
```

Comment:
We should probably change our target ubuntu version to the latest LTS. At the moment it's just using "latest", which might not actually be compatible with the latest LTS.